time.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 The GoPacket Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package gopacket
+
+import "fmt"
+
+// TimestampResolution represents the resolution of timestamps in Base^Exponent.
+type TimestampResolution struct {
```

Comment:
should this struct have a ToTime(int64) function attached to it?

Reply:
Hmm I'm not quiet sure if I understand correctly what this function should do.
Did you mean something, that returns `argument*TimestampResolution` as time.Time?
Hmm if I think more about this, this could actually only return a time.Duration, since the TimestampResolution does not specify an epoch (e.g., ntp timestamps use a different epoch than unix)

Reply:
I added a ToDuration function, tests and discovered a bug (milli vs micro was wrong...)